### DISCUSSION TITLE
Battery Level for Tuya Blind Motor (TS0601)

### DISCUSSION THREAD

**Discussion by westfa** [Koenkk/zigbee2mqtt#8632] [Devices]
Hi,

I have three Moes AM43 blind motors (zigbee_model=TS0601, zigbee_manufacturer=_TZE200_rddyvrci). They have batteries and I was wondering if it is possible to expose the battery level in addition to the state, position, motor speed and LQI?

Thanks

**Comment by nicknol** [comment]
the device status page shows the battery information, but it is not passed as an entity to Home Assistant.
